stable/metallb/templates/config.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if (or .Values.config .Values.arpCIDR) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
Just remove the namespace completely. It is unnecessary. The result will be the same. Apply everywhere.